site/features.md

```diff
@@ -55,6 +56,20 @@ now re-use the same tools and techniques when deploying applications
 as containers as we would have done when deploying them 'on metal' in
 our data centre.
 
+### <a name="optimised-traffic-transport"></a>Optimised traffic transport
```

Comment:
What's wrong with calling this section 'fast datapath'?  If that name is not suitable for a section title, we should find a name we like better and stick to it (fixing 'fastdp' in the status and WEAVE_NO_FASTDP accordingly).